src/serverdlg.h

```diff
@@ -90,9 +91,10 @@ public slots:
     void OnLocationCityTextChanged ( const QString& strNewCity );
     void OnLocationCountryActivated ( int iCntryListItem );
     void OnCentServAddrTypeActivated ( int iTypeIdx );
+    void OnNewRecordingClicked();
     void OnTimer();
-    void OnServerStarted() { UpdateSystemTrayIcon ( true ); }
```

Comment:
Oh, I see. You moved that to the cpp file. So you like it better to have it in the cpp?

Reply:
Well, there's _two_ lines of code now... It doubled in size!  I thought it might be better there.  If you're happy with it staying, I'll keep it in the header.

Reply:
Yes, that's ok. If it is more than one line, I also prefer to have it in the cpp file.